docs/pages/workflow/customizing.mdx

```diff
@@ -5,21 +5,19 @@ description: Learn how to add custom native code to your Expo project.
 
 import { Terminal } from '~/ui/components/Snippet';
 
-The Expo Go app is a great tool to get started. It exists to help developers quickly get projects off the ground, experiment with ideas (such as on [Snack](https://snack.expo.dev/)), and share their work with minimal friction. Expo Go makes this possible by including a feature-rich native runtime made up of every module in the [Expo SDK](/versions/latest/), so all you need to do to use a module is install the package and reload your app.
+Just as Expo projects can use third-party React Native packages containing native code, Expo projects are also ready for you to add your own custom native code to your project. Once you create your first [development build](/develop/development-builds/introduction/), you can use any native library or custom native code in your local development environment, as well as in your production app. Development builds work just like Expo Go, allowing you to quickly run your app from the [Expo CLI](/more/expo-cli/), except they include all your specific native dependencies and customizations, and not just what is included with the latest [Expo SDK](/versions/latest/).
 
-The tradeoff is that Expo Go does not allow you to add custom native code. You can only use native modules built into the Expo SDK. Many great libraries are available outside of the Expo SDK, and you may even want to build your native library. You can leverage these libraries with development builds or using prebuild to generate native projects, or both. You can also continue using [EAS Build](/build/introduction) to release your app as no changes are required.
+For most third-party native modules, thanks to [autolinking](/modules/autolinking/), all you need to do is install the package and create a new development build in order to start using the module's functionality. Some modules will also require you setup a [config-plugin](/config-plugins/introduction) to take care of other native customizations. For writing your own custom native code, there are different options depending on how you want to interact with native code in general.
```

Comment:
```suggestion
For most third-party native libraries, [autolinking](/modules/autolinking/) makes it easy. All you have to do is install the package and create a new development build to use the module. Some modules might also need a [config-plugin](/config-plugins/introduction) for additional native customizations. When adding your own custom native code, the approach varies based on how you want to work with native code.
```

- Avoid using words like thanks to
- Make it concise
- Fix CI warning

![CleanShot 2024-01-24 at 13 44 29@2x](https://github.com/expo/expo/assets/10234615/4b3c6e3d-20c9-46ee-bf05-f38721dd5d3e)